components/InfectionMedicalcareprovisionStatus.vue

```diff
@@ -0,0 +1,137 @@
+<template>
+  <div class="InfectionMedicalcareprovisionStatus">
+    <div class="InfectionMedicalcareprovisionStatus-heading">
+      <h3 class="InfectionMedicalcareprovisionStatus-title">
+        {{ $t('感染状況・医療提供体制（サマリ）') }}
+        {{ date }}時点
+      </h3>
+    </div>
+    <div class="InfectionMedicalcareprovisionStatus-Box">
+      <div class="InfectionMedicalcareprovisionStatus-Headline">
+        <app-link
+          to="https://www.fukushihoken.metro.tokyo.lg.jp/hodo/saishin/hassei.html"
+        >
+          {{ $t('感染状況') }}
+        </app-link>
+      </div>
+      <div class="InfectionMedicalcareprovisionStatus-description">
+        {{ $t('新規陽性者')
+        }}<span>{{ statuses.data['新規陽性者'].toLocaleString() }}人</span> /
+        {{ $t('検査数')
+        }}<span>{{ statuses.data['検査数'].toLocaleString() }}件</span>（{{
```

Comment:
```suggestion
        }}<span>{{ statuses.data['検査数'].toLocaleString() }}{{ $t('件.reports') }}</span>（{{
```

「件」については、`件.reports` で翻訳キーがあるようです。

Reply:
英語版の場合は単位を後ろにつけない方がいいのではという話もあり、（文字が長くなりすぎるというのもあり。）
ここで翻訳してしまうのいいのか、悩みどころです、

Reply:
おつかれさまです、
翻訳レビュー班のsisiです。

単位の翻訳不要だと思います。
タイトルに「者」とあるので、そのワードが各翻訳のタイトルに（英訳ならpatientなど）にもある場合は、単位は自ずと「人数」と理解されるので、表内やタイトルに続く数字（例えば973人→973）だけで良いかと。［人］は省略できるかと思います。
検査数も同じくタイトル翻訳に「数」（英訳ならPCRtestなど）があれば、［件］は省略で良いと考えますが、いかがでしょうか。

Reply:
@hello-iamsisi さん、フォローありがとうございます！なるほどですね、そうすると単位をつけるのは日本語のみということでよさそうです。

そうすると `v-if="['ja', 'ja-basic'].includes($i18n.locale)"` で出し分ける必要がありそうですね。

Reply:
こちら残課題で追って改修できたらと思います。

Reply:
これ、助数詞とか量詞といったものがあるかどうかという問題だと思うのですが、中国語・韓国語だと日本語に近いものが存在するようです。
`'en'`かどうかで判断する方が適切かもしれません。